HACKING.md

```diff
@@ -51,6 +51,8 @@ required.
 
 Scylla is built with [Ninja](https://ninja-build.org/), a low-level rule-based system. A Python script, `configure.py`, generates a Ninja file (`build.ninja`) based on configuration options.
 
+**Note**: On ubuntu it is possible that the installation directory will be under `/usr/bin/ninja` and CMake search this under `/usr/bin/ninja-build` so a possible sulotion can be using symlink. `ln -s /usr/bin/ninja /usr/bin/ninja-build`
```

Comment:
Regardless of your discussion with Avi below on what actually exists on Ubuntu (which I don't use), I don't understand what this new line is saying. /usr/bin/ninja is not an "installation directory", and our configure.py searches for either "ninja" or "ninja-build" in the user's path, so both should work just fine...

If anything, it's weird that HACKING.md keeps calling ninja with the command "ninja-build", when (as far as I know) it is available as "ninja" on all modern systems. A few years ago on Fedora you had to call it "ninja-build", but this is no longer true.